packages/vm-config/decentral-itest-vaults-config.json

```diff
@@ -104,6 +104,34 @@
         }
       ]
     },
+    {
```

Comment:
the benchmark shouldn't have to change any test config.

if it's really dependent on these changes, modify the structure after loading the JSON.

Reply:
Thanks for the additional context. I'd request that be in a separate request explaining that, when someone git blames the change, but i suppose that's low ROI

Reply:
Putting it in a separate PR might make sense, but since it was necessary to get my stuff to work it would necessarily be couple to this PR anyway, so, yeah, low ROI.